quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/util/SetKeepAliveTrueForTest.java

```diff
@@ -0,0 +1,13 @@
+package org.keycloak.it.cli.dist.util;
+
+import org.keycloak.it.utils.KeycloakDistribution;
+
+import java.util.function.Consumer;
+
+public class SetKeepAliveTrueForTest implements Consumer<KeycloakDistribution> {
```

Comment:
Isn't better to create a `@KeepServerAlive` annotation?

Reply:
done